src/main/java/com/minecolonies/coremod/colony/IColony.java

```diff
@@ -75,4 +75,16 @@
      * @return an integer with a describing value.
      */
     int getLastContactInHours();
+
+    /**
+     * Get the colony's raid index
+     * @return an integer describing the colony's raidlevel
+     */
+    int getRaidLevel();
+
+    /**
+     * get whether the colony will raid the following night or not
+     * @return a boolean value of whether the raid shall occur that night
+     */
+    boolean getWillRaid();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this method to start with "is" or "has". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2047)